shared/wallets/wallet-list/index.js

```diff
@@ -12,10 +19,11 @@ type AddProps = {
 
 const rowHeight = isMobile ? 56 : 48
 
-const styles = styleSheetCreate({
-  addContainerBox: {height: rowHeight, paddingTop: globalMargins.small},
-  progressIndicator: {height: 30, width: 30},
-})
+const HoverBox = isMobile
+  ? Kb.Box2
+  : glamorous(Kb.Box2)({
+      ':hover': {backgroundColor: globalColors.blueGrey2},
+    })
```

Comment:
You don't need to use `glamorous` for this. Add `className="hover_background_color_blueGrey2"` to `Kb.Box2` instead.